ts-test/src/App.tsx

```
@@ -0,0 +1,42 @@
+import './App.css'
+import { Button } from './components/ui/button'
+import { CardWithForm } from './Test'
+import { TabsDemo } from './Test2'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
+    <div className='items-center justify-center'>
+      <div>
+        <a href="https://vite.dev" target="_blank">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)